src/server/services/pipeline-in-progress/insertObviusData.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const moment = require('moment');
+const Meter = require('../../models/Meter');
+const Reading = require('../../models/Reading');
+const { log } = require('../../log');
+const demuxCsvWithSingleColumnTimestamps = require('./csvDemux');
+const loadArrayInput = require('./loadArrayInput');
+
```

Comment:
Add comment to this function and file

Reply:
Thanks for your comments! However, all the obvius-related files are merged directly from Nora's old-obvius branch for testing purpose only and will be removed soon. The complete version of them can be found in obvius2020.

Reply:
Thank you for the information